pkg/controller/oidcclientwatcher/oidcclientwatcher_controller.go

```diff
@@ -472,12 +472,15 @@ func (r *ReconcileOIDCClientWatcher) deploymentForOIDCClientWatcher(instance *op
 				ObjectMeta: metav1.ObjectMeta{
 					Labels: map[string]string{
 						"app": "oidcclient-watcher",
+						"app.kubernetes.io/instance": "common-iam",
 					},
 					Annotations: map[string]string{
 						"scheduler.alpha.kubernetes.io/critical-pod": "",
-						"productName":    "IBM Cloud Platform Common Services",
-						"productID":      "IBMCloudPlatformCommonServices_342_apache_0000",
-						"productVersion": "3.4.2",
+						"productName": "IBM Cloud Platform Common Services",
+						"productID": "068a62892a1e4db39641342e592daa25",
+						"productVersion": "3.3.0",
```

Comment:
is this the correct version?

Reply:
@yannizhang2019 Same version (3.3.0) is suggested in the epic.
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/35470#issuecomment-18320218